koku/masu/openapi.json

```diff
@@ -763,9 +773,8 @@
           "type": "object",
           "properties": {
             "api_version": {
-              "type": "integer",
-              "format": "int64",
-              "example": 1
+              "type": "string",
```

Comment:
This is actually returning a string instead of an integer, I changed this just to get the spec returning the data correctly but this probably needs fixing in masu to return the integer instead.